lib/worker.js

```diff
@@ -147,14 +142,17 @@ class AssetComputeWorker {
             this.cgroupSampler.start();
         }
 
-        this.directories = await createDirectories();
+        const folderName = '';
+        const baseDirectory = '';
+        const pipeline = false;
```

Comment:
For readability, I would rename this to:
```suggestion
        const usePipeline = false;
```